src/main/resources/application.properties

```diff
@@ -114,7 +114,7 @@ spring.mail.properties.mail.smtp.starttls.enable: false
 # end point for downstream and upstream search in
 # event repository
 er.url:
-#er.url: http://localhost:8080/search/
+#er.url: http://localhost:8080/eventrepository/search/
```

Comment:
I think url suffix varies depending on how ER is launched, with external tomcat or embedded webserver.
But keep the url that is most often valid.